tests/wdio/reports/result-de-chrome-terra-dev-site.json

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
Whoops, this looks like a bug that was recently logged/fix to terra toolkit, I'd manually remove these for now.

https://github.com/cerner/terra-toolkit-boneyard/pull/43

Reply:
I was wondering what those were lol. I'll have them out of there soon.

Reply:
Those reports have been removed.